ktor-features/ktor-auth-jwt/src/io/ktor/auth/jwt/JWTAuth.kt

```diff
@@ -118,8 +127,8 @@ private fun ApplicationRequest.parseAuthorizationHeaderOrNull() = try {
     null
 }
 
-private fun HttpAuthHeader.Companion.bearerAuthChallenge(realm: String): HttpAuthHeader =
-        HttpAuthHeader.Parameterized("Bearer", mapOf(HttpAuthHeader.Parameters.Realm to realm))
+private fun HttpAuthHeader.Companion.bearerAuthChallenge(realm: String, supportedAuthSchemes: Set<String>): HttpAuthHeader =
+        HttpAuthHeader.Parameterized(supportedAuthSchemes.first(), mapOf(HttpAuthHeader.Parameters.Realm to realm))
```

Comment:
it looks like we need to produce multiple auth challenges, not just for the first one